recipe/meta.yaml

```diff
@@ -1,31 +1,28 @@
 {% set name = "pydeck" %}
 {% set version = "0.7.1" %}
+{% set sha256 = "907601c99f7510e16d27d7cb62bfa145216d166a2b5c9c50cfe2b65b032ebd2e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 907601c99f7510e16d27d7cb62bfa145216d166a2b5c9c50cfe2b65b032ebd2e
+  sha256: {{ sha256 }}
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
+  # noarch: python
 
 requirements:
   host:
-    - python >=3.6
-    - ipykernel
-    - ipython
-    - jinja2
+    - python 
     - pip
-    - python
-    - traitlets
-    - numpy >=1.16.4
+    - setuptools <61
+    - wheel
   run:
```

Comment:
We need to update run dependencies, see https://github.com/visgl/deck.gl/blob/6708cd139680863d17335600066257a2d16c2c68/bindings/pydeck/setup.py#L213-L219

Reply:
thank you! I'll start doing this more deliberately now that I understand that.